experiments/Cargo.toml

```diff
@@ -10,8 +10,8 @@ license = "MPL-2.0"
 keywords = ["experiment"]
 
 [lib]
-name = "experiments"
-crate-type = ["lib"]
+name = "nimbus"
```

Comment:
This sets the name of the generated `.so` file, which the android component assumes will be "nimbus".